client/blocks/reader-subscription-list-item/style.scss

```diff
@@ -12,7 +12,7 @@
 .reader-subscription-list-item__byline {
 	color: darken( $gray, 30% );
 	margin-right: 16px;
-	flex: 1 1 auto;
+	flex: 1 1 0px;
```

Comment:
- For site descriptions that have a URL, its container adapts to content size (auto) hence it was expanding really wide in IE so we're setting the flex-basis initial value at 0.
- Adding `px` because IE doesn't interpret flex-basis without a unit very well.